### PR TITLE
Added handlebars plugin

### PR DESCRIPTION
**Pull request by Inqnuam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,4 +175,4 @@ Before creating a PR, please make sure that:
 * You are the author of the plugin
 * You are willing to respond to issues about the plugin if people file issues against it
 
-_Note: Even though the text "esbuild-plugin-example" above is a link, it's just example placeholder text. The project at the link destination intentionally doesn't exist. The text is only linked to demonstrate how you should format your entry._
+_Note: Even though the text "esbuild-plugin-example" above is a link, it's just example placeholder text. The project at the link destination intentionally doesn't exist. The text is only linked to demonstrate how you should format your entry._
```
Comment (evanw) [inline README.md:178]: This line is unrelated to your plugin and should not have been changed.

Comment (Inqnuam) [reply README.md:178]: Yes, I noticed that but I can't see what's the difference

Comment (evanw) [reply README.md:178]: I assume your editor is configured to strip the trailing newline.
